README.md

```diff
@@ -10,6 +10,7 @@ IRC channel: #mobile.
 
 Before trying to make edits via pull requests:
 * Get in touch with the [existing localization team](https://wiki.mozilla.org/L10n:Teams).
+    * If your PR touches many files, it will get stale very quickly because many contributors add to these files. Please make sure there is someone around to merge your PR in a timely manner, or expect to need to update your PR to resolve conflicts.
```

Comment:
I'm not sure this is the right place to add this information, these two bullet points are targeting localizers. Maybe better a section "For developers"?

Reply:
@flodolo good point! Sorry for the delayed response, was away, but I've updated the PR with a new section.